apps/marketplace/components/BuyerRFX/BuyerRFXAdditionalInformationStage.js

```diff
@@ -2,19 +2,45 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import { actions, Form } from 'react-redux-form'
-import { required, validPhoneNumber, dateIs2DaysInFuture } from 'marketplace/components/validators'
 import formProps from 'shared/form/formPropsSelector'
 import Textfield from 'shared/form/Textfield'
 import AUheading from '@gov.au/headings/lib/js/react.js'
+import format from 'date-fns/format'
+import addDays from 'date-fns/add_days'
+import {
+  required,
+  validPhoneNumber,
+  validDate,
+  dateIs2DaysInFuture,
+  dateIsBefore
+} from 'marketplace/components/validators'
 import ErrorAlert from 'marketplace/components/Alerts/ErrorAlert'
-import ClosingDateControl from 'marketplace/components/BuyerBriefFlow/ClosingDateControl'
+import DateControl from 'marketplace/components/BuyerBriefFlow/DateControl'
 import CheckboxDetailsField from 'shared/form/CheckboxDetailsField'
-import styles from './BuyerRFXAdditionalInformationStage.scss'
+import styles from '../BuyerSpecialist/BuyerSpecialistAdditionalInformation.scss'
 
-class BuyerRFXAdditionalInformationStage extends Component {
+const requiredContactNumber = v => required(v.contactNumber)
+const contactNumberFormat = v => validPhoneNumber(v.contactNumber)
+const requiredClosedAt = v => required(v.closedAt)
+const closedAtIsValid = v => validDate(v.closedAt)
+const closedAtIs2DaysInFuture = v => !closedAtIsValid(v) || dateIs2DaysInFuture(v.closedAt)
+const closedAtIsBefore = v => !closedAtIsValid(v) || dateIsBefore(v.closedAt, addDays(new Date(), 366))
```

Comment:
Is 365 the correct value to use here if our rule is that the date must be no more than one year from now? 
```suggestion
const closedAtIsBefore = v => !closedAtIsValid(v) || dateIsBefore(v.closedAt, addDays(new Date(), 365))
```